stanford/bin/dump-logs

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+# Usage: dump-logs [/edx/var|<output_directory>]
```

Comment:
@kluo You can now set a custom output directory, if you want it somewhere else, presumably where you already have write access. The default is still `/edx/var`.

```sh
dump-logs ~/backup-cme
```
and you can configure which deployments/clusters to target with environment variables:

```sh
DUMP_LOGS_DEPLOYMENTS=cme DUMP_LOGS_CLUSTERS=certs dump-logs ~/backup-cme
```